woocommerce/woocommerce
# CFE/ Revert #53544 and #53548

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Reverts #53544 and #53548 to resolve label inconsistencies caused by updates enabling merchants to edit “Subtotal” and “Total” headings. These changes disrupted recurring product detail sections when using WooCommerce Subscriptions.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

For cart and checkout:

1. Edit the cart and checkout pages. Check the “Subtotal” and “Total” headings in the order summary section.
2. Confirm that editing the text directly for either heading no longer reflects changes (functionality should be rolled back).
3. Save the changes and confirm no unexpected errors appear.
4. Ensure both “Subtotal” and “Total” headings revert to their original default states on the frontend.

For translations:

1. Install translations for Nederlands while keeping the admin language in English.
2. Verify that the “Subtotal” and “Total” headings default to the correct translations (“Subtotaal” and “Totaal”) on the frontend.
3. Swap between languages in WordPress and ensure consistency in the translated text without customizations.



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
